src/containers/App/App.test.js

```diff
@@ -18,6 +18,11 @@ import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 
 import { App } from './App';
+import * as API from '../../api';
+
+beforeEach(() => {
```

Comment:
Because we get this warning in the tests without it: `console.warn node_modules/fetch-mock/src/lib/fetch-handler.js:73
    Unmatched get to http://localhost/v1/namespaces/undefined/pipelines/` All of the App.js tests use the `getPipelines` function, so I wanted to apply the mock function to all of the tests.

If you'd prefer I apply the mock function individually in each test function, I can make that change.